projects/fsastorefrontlib/src/cms-components/assets-table/assets-table.component.html

```diff
@@ -0,0 +1,53 @@
+<ng-container *ngIf="assets?.length > 0; else noAssets">
+  <table class="box-shadow table table-hover border border-color-3 mt-5">
+    <thead class="d-none d-md-table-row">
+      <th *ngFor="let heading of headings" scope="col">{{ heading }}</th>
+    </thead>
+    <tbody>
+      <tr class="position-relative" *ngFor="let asset of assets">
+        <td class="d-flex d-md-table-cell">
+          <div class="mobile-label text-uppercase pr-2 d-md-none">
+            {{ headings[0] }}:
```

Comment:
This should be refactored as part of this [ticket](https://sapjira.wdf.sap.corp/browse/CXFSA-412)